gems/aws-sdk-sqs/lib/aws-sdk-sqs/queue_poller.rb

```diff
@@ -392,17 +390,18 @@ def send_request(config)
         @client.receive_message(params)
       end
 
-      def check_idle_timeout(config, stats, messages)
-        if config.idle_timeout
-          since = stats.last_message_received_at || stats.polling_started_at
-          idle_time = Time.now - since
-          throw :stop_polling if idle_time > config.idle_timeout
-        end
+      def check_idle_timeout(config, stats, _messages)
```

Comment:
Since this is a private method - I think we can just remove this parameter

Reply:
Double check it's not called anywhere else...